google/cloud/forseti/scanner/scanner_builder.py

```diff
@@ -54,10 +54,16 @@ def build(self):
         for scanner in self.scanner_configs.get('scanners'):
             if scanner.get('enabled'):
                 module_path = 'google.cloud.forseti.scanner.scanners.{}'
-                module_name = module_path.format(
-                    scanner_requirements_map.REQUIREMENTS_MAP
-                    .get(scanner.get('name'))
-                    .get('module_name'))
+
+                scanner_name = scanner.get('name')
+                scanner_def = scanner_requirements_map.\
+                    REQUIREMENTS_MAP.get(scanner_name)
+                # check if the scanner exists
+                if not scanner_def:
+                    LOGGER.warn('Scanner is undefined')
```

Comment:
We should log with the actual name of the scanner, and I would consider the error level to be more appropriate.
```
LOGGER.error('Configured scanner is undefined: %s, scanner.get('name'))
```

Reply:
modified as suggested